packaging/hoodie-spark-bundle/pom.xml

```diff
@@ -247,21 +247,25 @@
       <groupId>${hive.groupid}</groupId>
       <artifactId>hive-service</artifactId>
       <version>${hive.version}</version>
+      <scope>runtime</scope>
```

Comment:
what does scope runtime mean please ?

Reply:
@n3nash : Thanks for the question. This should actually be "compile" time scope. I was testing some variations in scope and ended up with "runtime".